### DISCUSSION TITLE
layout.base was not found.

### DISCUSSION THREAD

**Discussion by pickluA** [livewire/livewire#2998] [7. Bugs (archived)]
### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: [e.g. 2.2.3]
- Laravel version: [e.g. 8.3.1]
- Alpine version: [e.g. 2.3.0]
- Browser: [e.g. Chrome, Safari]

**Comment by joshhanley** [comment]
@pickluA please provide some more information and code that you have that is causing the issue.

Thanks!
